adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryExecParamRepository.java

```diff
@@ -235,20 +235,20 @@ public String getExecParamValue(final String key) throws IllegalArgumentExceptio
      */
     @Override
     public Options getOptions() {
-        final Options options = new Options();
-        options.addOption(String.valueOf(URL_KEY.charAt(0)), URL_KEY, true,
-                "URL to GTFS zipped archive");
-        options.addOption(String.valueOf(ZIP_KEY.charAt(0)), ZIP_KEY, true,
+        final Options options = new Options()
```

Comment:
Those changes don't seem to be related to the PR subject

Reply:
Indeed, reverting these changes. Will open a new issue to use the chained method addOption